libs/standard-tests/langchain_standard_tests/unit_tests/chat_models.py

```diff
@@ -21,6 +21,12 @@ def my_adder_tool(a: int, b: int) -> int:
     return a + b
 
 
+@tool
+def magic_function(input: int) -> int:
```

Comment:
whats this for?

Reply:
oh i see, but why does it have to be defined in unit test?

Reply:
I just put it there because that's where the other tool was. I moved it to the integration test module.